wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWComponent.java

```diff
@@ -1028,7 +949,7 @@ private static boolean isFlagSet(final int flags, final int mask) {
 	public boolean hasTabIndex() {
 		return false;
 	}
-
+        
```

Comment:
Revert. It appears you have not set your IDE to use tab indent for this project, please refer to our code standards which require tabs for Java.